tools/mo/unit_tests/mo/ops/elementwise_test.py

```diff
@@ -116,6 +121,10 @@ def test_first_input_const(self):
         const_node = Node(graph, 'const')
         self.assertEquals(const_node.out_port(0).get_data_type(), np.float32)
```

Comment:
```suggestion
        self.assertEqual(const_node.out_port(0).get_data_type(), np.float32)
```

Reply:
and in three other tests just need to change assertEquals -> assertEqual

Reply:
Seems like it helped, thanks, I'll close the corresponding ticket.